.github/workflows/linters.yml

```diff
@@ -0,0 +1,48 @@
+name: Linters
+
+on: pull_request
+
+env:
+  FORCE_COLOR: 1
+
```

Comment:
- [ Optional]  Please use present tense to write your commits messages 
![image](https://user-images.githubusercontent.com/74744787/164073012-65c89265-93cf-4413-8a49-95d2c8bff122.png)

Reply:
Noted, thanks